management/azure_mgmt_resources/spec/resource_groups_spec.rb

```diff
@@ -68,7 +68,7 @@
     params = Models::ResourceGroup.new
     expect{@client.update(nil, params)}.to raise_error(ArgumentError)
     expect{@client.update('foo', nil)}.to raise_error(ArgumentError)
-    expect{@client.update('~`123', params).value!}.to raise_error(Faraday::SSLError)
+    expect{@client.update('~`123', params).value!}.to raise_error(MsRestAzure::AzureOperationError)
```

Comment:
If I run this locally on my machine, I get Faraday::SSLError. But, in the CI machine, I get MsRestAzure::AzureOperationError. Any idea??????